oap-stdlib/src/main/java/oap/http/server/nio/NioHttpServer.java

```diff
@@ -151,31 +184,68 @@ private void startNewPort( int port ) {
-                Metrics.gauge( "nio_requests", Tags.of( "port", sPort, "type", "errors" ), connectorStatistics, ConnectorStatistics::getErrorCount );
+                Metrics.gauge( "nio_requests", Tags.of( "port", sPort, "schema", schema.name(), "type", "total" ), connectorStatistics, ConnectorStatistics::getRequestCount );
+                Metrics.gauge( "nio_requests", Tags.of( "port", sPort, "schema", schema.name(), "type", "active" ), connectorStatistics, ConnectorStatistics::getActiveRequests );
+                Metrics.gauge( "nio_requests", Tags.of( "port", sPort, "schema", schema.name(), "type", "errors" ), connectorStatistics, ConnectorStatistics::getErrorCount );
 
-                Metrics.gauge( "nio_connections", Tags.of( "port", sPort, "type", "active" ), connectorStatistics, ConnectorStatistics::getActiveConnections );
+                Metrics.gauge( "nio_connections", Tags.of( "port", sPort, "schema", schema.name(), "type", "active" ), connectorStatistics, ConnectorStatistics::getActiveConnections );
 
-                Metrics.gauge( "nio_pool_size", Tags.of( "port", sPort, "name", "worker", "type", "active" ), server, s -> s.getWorker().getMXBean().getWorkerPoolSize() );
-                Metrics.gauge( "nio_pool_size", Tags.of( "port", sPort, "name", "worker", "type", "core" ), server, s -> s.getWorker().getMXBean().getCoreWorkerPoolSize() );
-                Metrics.gauge( "nio_pool_size", Tags.of( "port", sPort, "name", "worker", "type", "max" ), server, s -> s.getWorker().getMXBean().getMaxWorkerPoolSize() );
-                Metrics.gauge( "nio_pool_size", Tags.of( "port", sPort, "name", "worker", "type", "busy" ), server, s -> s.getWorker().getMXBean().getBusyWorkerThreadCount() );
-                Metrics.gauge( "nio_pool_size", Tags.of( "port", sPort, "name", "worker", "type", "queue" ), server, s -> s.getWorker().getMXBean().getWorkerQueueSize() );
+                Metrics.gauge( "nio_pool_size", Tags.of( "port", sPort, "schema", schema.name(), "name", "worker", "type", "active" ), server, s -> s.getWorker().getMXBean().getWorkerPoolSize() );
+                Metrics.gauge( "nio_pool_size", Tags.of( "port", sPort, "schema", schema.name(), "name", "worker", "type", "core" ), server, s -> s.getWorker().getMXBean().getCoreWorkerPoolSize() );
+                Metrics.gauge( "nio_pool_size", Tags.of( "port", sPort, "schema", schema.name(), "name", "worker", "type", "max" ), server, s -> s.getWorker().getMXBean().getMaxWorkerPoolSize() );
+                Metrics.gauge( "nio_pool_size", Tags.of( "port", sPort, "schema", schema.name(), "name", "worker", "type", "busy" ), server, s -> s.getWorker().getMXBean().getBusyWorkerThreadCount() );
+                Metrics.gauge( "nio_pool_size", Tags.of( "port", sPort, "schema", schema.name(), "name", "worker", "type", "queue" ), server, s -> s.getWorker().getMXBean().getWorkerQueueSize() );
 
             }
         }
     }
 
+    private synchronized SSLContext getSSLContext() {
```

Comment:
I do not think that you need to create a SSLContext as you can just pass the manager to `builder.addHttpsListener( port, "0.0.0.0", managers, null, handler )`